policy/policy.go

```diff
@@ -42,17 +42,20 @@ var (
 	// EmptyPolicy represents an empty policy.
 	EmptyPolicy Policy
 
-	// EmptyVersionedPolicies represents an empty VersionPolicies.
-	EmptyVersionedPolicies VersionedPolicies
+	// EmptyStagedPolicies represents an empty staged policies.
+	EmptyStagedPolicies StagedPolicies
 
-	errNilPolicySchema = errors.New("nil policy schema")
+	// DefaultPoliciesList represents a default policies list.
+	DefaultPoliciesList = PoliciesList{EmptyStagedPolicies}
 
 	// defaultPolicies are the default policies.
 	// TODO(xichen): possibly make this dynamically configurable in the future.
 	defaultPolicies = []Policy{
```

Comment:
Hey I think we definitely need to make these default policies configurable ASAP.  Otherwise if we don't and we need to make an architectural change to the code to make it configurable later it will be ignored and not done and then makes it a lot less useful to others.